src/objects/Mesh.js

```diff
@@ -206,7 +206,7 @@ THREE.Mesh.prototype.raycast = ( function () {
 						distance: distance,
 						point: intersectionPoint,
 						face: new THREE.Face3( a, b, c, THREE.Triangle.normal( vA, vB, vC ) ),
-						faceIndex: null,
+						faceIndex: i/3,
```

Comment:
`face.a / 3` is already available to you.

Quite frankly, I think that returning a `THREE.Face3` with those parameters is somewhat misleading. I'm not sure of a better solution, however.